bin/v-add-fastcgi-cache

```diff
@@ -36,6 +38,16 @@ is_object_unsuspended 'user' 'USER' "$user"
 is_object_valid 'web' 'DOMAIN' "$domain"
 is_object_unsuspended 'web' 'DOMAIN' "$domain"
 
+if ! [[ "$duration" =~ ^[0-9].*[s|m|d]$ ]]; then
```

Comment:
Currently accepts:
10m 10s and 10d according what is allowed but also accepts 
10ms 10sd or 10dm if somebody know how to limit it